docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -308,101 +309,174 @@ Here is a sample of this configuration file:
+
+``ec2_build_farm (AWSEC2)``
+"""""""""""""""""""""""""""
+
+.. literalinclude:: /../deploy/sample-backup-configs/sample_config_build_farm.yaml
+   :language: yaml
+   :start-after: DOCREF START: EC2 Build Farm
+   :end-before: DOCREF END: EC2 Build Farm
+
+This build farm configuration provides the default EC2 build farm functionality
+of FireSim 1.13.x and earlier. For each AGFI (FireSim AWS EC2 F1 FPGA Image) being
+built, a new EC2 instance (build host) will be spawned according to the build farm configuration
+arguments. Its ``build_farm_type`` is set as the ``AWSEC2`` build farm class.
+
+The ``instance_type`` argument defines the type of instance that the build will run on.
+Generally, running on a ``z1d.2xlarge`` is sufficient. In our experience, using more
+powerful instances than this provides little gain.
+
+The ``build_instance_market`` argument specifies the AWS market used. You can specify either
+``spot`` or ``ondemand`` here.
+
+When ``build_instance_market: spot``, the ``spot_interruption_behavior`` and ``spot_max_price``
+arguments determines spot instance behaviors. The ``spot_interruption_behavior`` value
+determines what happens to an instance if it receives the interruption signal from AWS. You can specify
+either ``hibernate``, ``stop``, or ``terminate`` for this value. The ``spot_max_price``
+value determines the max price you are willing to pay per instance, in dollars.
+You can also set it to ``ondemand`` to set your max to the on-demand price for the instance.
+
+The ``default_build_dir`` argument specifies the default location of the build directory used
+on the spawned build host which will hold build outputs from Vivado and other tools.
+
+``local_build_farm (ExternallyProvisioned)``
+""""""""""""""""""""""""""""""""""""""""""""
+
+.. literalinclude:: /../deploy/sample-backup-configs/sample_config_build_farm.yaml
+   :language: yaml
+   :start-after: DOCREF START: Local Build Farm
+   :end-before: DOCREF END: Local Build Farm
+
+This build farm configuration allows users to provide their own build hosts
+(build machines) to be used for building FPGA images.
+In the case of AGFI builds and this specific configuration, this allows for users to build AGFIs locally
+(and potentially on other build hosts).
+Its ``build_farm_type`` is set as the ``ExternallyProvisioned`` build farm class.
+This build farm class assumes that the list of IPs given to it are pre-setup to work
+with FireSim (Vivado installed, FireSim dependencies installed, etc).
+
+The ``default_build_dir`` argument specifies the default location of the build directory used
+on the build host which will hold build outputs from Vivado and other tools.
+This location can be overridden by specific build farm hosts (see ``build_farm_hosts`` description).
+
+The ``build_farm_hosts`` argument section provides a list of IP addresses (and ``localhost``) that
```

Comment:
```suggestion
The ``build_farm_hosts`` argument section provides a list of IP addresses that
```
The "and" was confusing and your example covers it.

Reply:
also you probably want to use "host names" in place of "IP addresses" in this section.